github/repos.go

```diff
@@ -538,6 +540,12 @@ type RequiredPullRequestReviews struct {
 	IncludeAdmins bool `json:"include_admins"`
```

Comment:
I'll contact support and try out how API behaves currently if no value is set for `RequiredPullRequestReviews`